gem/styles/gem-rwanda/modules/_m-article-teasers-tags.scss

```diff
@@ -25,6 +25,11 @@
     font-size: $font-small;
     text-decoration: underline;
     text-underline-position: under;
+    .base-teaser__title{
```

Comment:
@sewagodimo Please locate which UI that has the classes - article teasers tags. 
And then check where this element picking up these styles rules